llvm/lib/Target/X86/X86MCInstLower.cpp

```diff
@@ -2538,26 +2539,6 @@ void X86AsmPrinter::emitInstruction(const MachineInstr *MI) {
 
   case X86::SEH_BeginEpilogue: {
```

Comment:
Can this be folded into the case statement above, or will they fail on this assert?

Reply:
Maybe, but I don't know the invariants well enough to say whether that assert is appropriate for all of the `SEH_*` ops.